### DISCUSSION TITLE
What is wrong with all of you?!?

### DISCUSSION THREAD

**Discussion by NubeNueve** [community/community#113734] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have been trying to figure out how to use GitHub and contribute correctly for years now and everytime  I try to, everyone keeps blocking me! like I’m tired of this. I have a lot to contribute and I have really good ideas and you guys are holding me back from becoming a developer.

**Comment by ghost** [comment]
Me on the other hand, I don't have anything contribute, no good ideas, I'm basically here to take what I can and split. So nice to meet everybody on this chatroom.

**Comment by vickkie** [comment]
Maybe tell people you are a Hip hop artist and composer 😂😂
it might work

**Reply by NubeNueve** [reply]
Stfu
